cmd/main.go

```diff
@@ -203,11 +206,27 @@ func (c Configuration) isFileToBeExcluded(file string, existingConsumersFiles ma
 	return isFileExcluded || existingConsumersFiles[file]
 }
 
+// Gets the full list of paths to monitor
+func (c Configuration) getCompleteListOfPaths(pathList []string) []string {
+	logger := c.logger()
+	var completePathList []string
+	for _, path := range pathList {
+		completePath, err := filepath.Glob(path)
+		if err != nil {
+			logger.Error().Err(err).Msgf("Error getting complete list of paths to register: %v", err)
```

Comment:
probably "continue" should follow otherwise you add to completePathList something unexpected

Reply:
updated